code/modules/mob/living/simple_animal/shade.dm

```diff
@@ -30,10 +30,12 @@
 	flying = TRUE
 	meat_type = /obj/item/weapon/ectoplasm
 	mob_property_flags = MOB_SUPERNATURAL
+	var/given_cult_language = 1 //1 means they will gain the cult language, 0 means they won't
```

Comment:
Don't do it like that. Override the children to remove the cult language if they don't need.